src/custom/constants/GPv2Settlement.ts

```diff
@@ -0,0 +1,13 @@
+import { ChainId } from '@uniswap/sdk'
+import { Interface } from '@ethersproject/abi'
+// Can even prune ABI down to onlywhat we use
+// like events only
+import GP_V2_SETTLEMENT from './GPv2Settlement.json'
```

Comment:
Yes, I would do that. Our use of it is minimal, I wouldn't add extraweight to the web if we can avoid it